toolbox/gpu-operator/operand-build/operands/dcgm_exporter.json

```diff
@@ -0,0 +1,23 @@
+{
+  "build_operand_git_repo": "https://gitlab.com/nvidia/container-toolkit/gpu-monitoring-tools",
```

Comment:
We might want to fork all the operand repos, to be more in a control.

For example, all the changes we might want to make to support different architectures seamlessly - it might be annoying to wait changes like that to be merged in nvidia upstream.

Reply:
Also, when they're forked - they're less of a moving target, we're more in control of which versions we are testing

Reply:
we don't *want* to fork them, but we may *have* to do so :)
it the image cannot be easily built for an architecture and the dockerfile file needs to be fixed, fork the repo and fix what ever has to be fixed; (make sure we agree on the fix) and share the PR with NVIDIA, so that we can get rid of our fork

> Also, when they're forked - they're less of a moving target, we're more in control of which versions we are testing

one thing to take into account somehow it that a given version of the GPU Operator works with a given version of the operands. There might be backward compatibility, but not necessarily forward.

So we need some flexibility in the configuration of the `build_operand_git_ref`, and maybe in the `build_operand_tag`.